zipkin-scrooge/build.sbt

```diff
@@ -1,11 +1,4 @@
-import com.twitter.scrooge.ScroogeSBT
-
-defaultSettings ++ ScroogeSBT.newSettings
-
-ScroogeSBT.scroogeThriftSourceFolder in Compile <<= (
-  (baseDirectory in ThisBuild)
-  (_ / "zipkin-thrift" / "src" / "main" / "thrift" / "com" / "twitter" / "zipkin" )
-)
+scroogeThriftSourceFolder in Compile := file("./zipkin-thrift/src/main/thrift")
```

Comment:
that syntax doesn't work, but it also doesn't work for unrelated projects. I'm hesitant to jump down more build areas, but if you think this is important let's at least log an issue.

Doesn't seem related, or a feature we use in travis or the release process. That's why I think we can separate out "sbt project X" from this change, and from the general work on getting the build green.